src/utils/queries.ts

```diff
@@ -27,12 +27,14 @@ export const buildUserFetcher = (user: User) => {
         } as FetchedUser;
     };
 };
-type FetchedUserProfile = {
+export type FetchedUserProfile = {
     username: string;
     email: string;
     displayName: string;
     bio: string;
     status: UserStatus;
+    following: number;
+    followers: number;
```

Comment:
prolly better to have these as numFollowing, numFollowers so we don't get mixed up thinking these are lists of Users

Reply:
True, just pushed the change